src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ExtensionDataObject.cs

```diff
@@ -37,23 +37,20 @@ public class ExtensionDataMember
     internal class ExtensionDataMember
 #endif
     {
-        private string _name;
-        private string _ns;
-        private IDataNode _value;
+        private IDataNode? _value;
         private int _memberIndex;
-        public string Name
-        {
-            get { return _name; }
-            set { _name = value; }
-        }
 
-        public string Namespace
+        public ExtensionDataMember(string name, string ns)
```

Comment:
this would be a new API if USE_REFEMIT is defined

Reply:
USE_REFEMIT is never defined. Will be removed with #41512.